src/subdomains/core/aml/enums/aml-error.enum.ts

```diff
@@ -19,8 +19,8 @@ export enum AmlError {
   NO_LETTER = 'NoLetter',
   NO_AML_LIST = 'NoAmlList',
   NO_KYC_FILE_ID = 'NoKycFileId',
-  NO_NAME_CHECK = 'NoNameCheck',
-  OUTDATED_NAME_CHECK = 'OutdatedNameCheck',
+  NAME_CHECK_WITHOUT_KYC = 'NameCheckWithoutKYC',
+  NAME_CHECK_DESPITE_BIRTHDAY = 'NameCheckDespiteBirthday',
```

Comment:
Should we name it `NAME_CHECK_WITH_BIRTHDAY`? Would fit better to `NAME_CHECK_WITHOUT_KYC` 😁